.github/workflows/checks-and-tests.yml

```diff
@@ -1,5 +1,8 @@
 name: Checks and Tests
-on: [push, workflow_dispatch]
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+  workflow_dispatch:
```

Comment:
This will skip drafts?

Reply:
Oh I see, I guess that's fine.

Originally I was just interested in what the types meant :smile: